practical/groovy/analyse_image_map_annotation.groovy

```diff
@@ -63,17 +69,8 @@ import ij.IJ
 import ij.plugin.frame.RoiManager
 import ij.measure.ResultsTable
 
-// Setup
-// =====
-
-// OMERO Server details
-HOST = "workshop.openmicroscopy.org"
-PORT = 4064
 group_id = "-1"
-// parameters to edit
-image_id = 1001
-USERNAME = "username"
-PASSWORD = "password"
+
```

Comment:
This script works fine. It would be cool if we could harvest the version of Fiji in https://github.com/ome/training-scripts/pull/78/files#diff-7f5412542070ee2bdcdb85522dac22c6R137 and put it into the KVPs - just an RFE

Reply:
I could look into that in a follow-up PR